frontend/app/src/types/asset/asset-urls.ts

```diff
@@ -87,6 +82,11 @@ export const explorerUrls: AssetExplorerUrls = {
     transaction: 'https://scrollscan.com/tx/',
     block: 'https://scrollscan.com/block/',
   },
+  [Blockchain.ZKSYNC_LITE]: {
+    address: 'https://zkscan.io/explorer/accounts/',
+    transaction: 'https://zkscan.io/explorer/transactions/',
+    block: 'https://zkscan.io/explorer/blocks/',
+  },
```

Comment:
Great, can you create a note or something? It would make sense to expose it through the api rather than having it repeated in both backend end frontend.